src/packages/solid/.storybook/preview.tsx

```diff
@@ -40,6 +40,7 @@ export const args = {
 
 const preview = {
   parameters: {
+    backgrounds: { default: 'black' }, // sets background dark
```

Comment:
sets background color for all stories to black like we have setup in LUI

Reply:
this didnt seem to work?

Reply:
hmm, I'll take a look again. I can always create a separate PR to adjust the Storybook theme, not a priority item for sure

Reply:
either way works for me!